### ISSUE TITLE
Wrong example of SSHD usage in README.md

### ISSUE THREAD

**Issue by Suseika** [jcabi/jcabi-ssh#24]
Readme suggests using `SSHD` class like this:

```java
try (SSHD sshd = new SSHD()) {
  String uptime = new Shell.Plain(
    SSH(sshd.host(), sshd.login(), sshd.port(), sshd.key())
 ).exec("uptime");
}
```

However, `SSHD() ` no-arguments constructor doesn't exist. The only constructor is `SSHD(File path)`.

Either the readme has to be fixed, or the consturctor implemented.

**Comment by dmarkov** [comment]
@yegor256 please do something about this issue

**Comment by yegor256** [comment]
@Suseika fixed in 34f440f
